File: bm_ui_xam_forms/bm_ui_xam_forms/bm_ui_xam_forms/Model/Information.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace bm_ui_xam_forms.Model
{
    public class Information
    {
        public string Start { get; set; }
        public string End { get; set; }

        public Information(string start, string end)
        {
            this.Start = start;
            this.End = end;
        }

    }
}

File: bm_ui_android_kotlin/app/src/main/java/de/colinsteffen/bm_ui_android_kotlin/Model/Information.java
package de.colinsteffen.bm_ui_android_kotlin.Model;

public class Information {
    private String start;
    private String end;

    public Information(String start, String end) {
        this.start = start;
        this.end = end;
    }

    public String getStart() {
        return start;
    }

    public String getEnd() {
        return end;
    }

    public void setStart(String start) {
        this.start = start;
    }

    public void setEnd(String end) {
        this.end = end;
    }

}
